app/components/ui-table/cell/cell-date.hbs

```diff
@@ -1 +1 @@
-{{general-date @record @props.options.timezone @props.options.dateFormat}}
+{{general-date @record @props.options.timezone '' @props.options.dateFormat}}
```

Comment:
Why the empty string?

Reply:
sorry, not required here, will revert back.